src/main/kotlin/uk/gov/justice/digital/hmpps/hmppssubjectaccessrequestworker/services/GeneratePdfService.kt

```diff
@@ -142,10 +142,10 @@ class GeneratePdfService {
   fun addData(pdfDocument: PdfDocument, document: Document, services: List<DpsService>) {
     services.forEach { service ->
       document.add(AreaBreak(AreaBreakType.NEXT_PAGE))
-      log.info("Compiling data from " + service.businessName)
+      log.info("Compiling data from ${service.businessName ?: service.name}")
 
       val renderedTemplate = templateRenderService.renderTemplate(serviceName = service.name!!, serviceData = service.content)
```

Comment:
The check should really be before this line, so it doesn't bother wasting time rendering the templates if it doesn't have to.